ansible/roles/jenkins-master/defaults/main.yaml

```diff
@@ -36,3 +36,5 @@ jenkins_plugins:
   - timestamper
```

Comment:
Regarding commit longer msg, we do not *need* this. I suggest instead:

=> We need this to allow the GitHub API wrapper to communicate with GitHub API without duplicating the credentials in Jenkins and in the filesystem.

Side comment: I'd prefer to use only this GitHub API wrapper instead of doing half of things using it and another half using GitHub Pull Request Builder Jenkins plugin, but the opinion of the rest of the team may differ.